src/points.jl

```diff
@@ -28,7 +28,7 @@ end
 Point{Dim,T}(coords...) where {Dim,T} = Point{Dim,T}(SVector{Dim,T}(coords...))
 Point(coords::SVector{Dim,T}) where {Dim,T} = Point{Dim,T}(coords)
 Point(coords::AbstractVector{T}) where {T} = Point{length(coords),T}(coords)
-Point(coords...) = Point(SVector(coords...))
+Point(coords...) = Point(float(SVector(coords...)))
```

Comment:
What we really want is a more specific method that dispatches on `Integer` and converts to `Float64`, otherwise will be ignoring user input. Only the specific case where the user types integer literals needs to be amended.

Reply:
> What we really want is a more specific method that dispatches on Integer and converts to Float64

Sorry, but I could not get what you really want. 

> Only the specific case where the user types integer literals needs to be amended.

This is just the case. Here are sample constructors. 
```julia 
julia> Point(1, 2) 
Point(1.0, 2.0)

julia> Point(1., 2.)
Point(1.0, 2.0)

julia> Point(1f0, 2f0) 
Point(1.0f0, 2.0f0)

ulia> Point([1, 2])
Point(1, 2)

julia> Point([1., 2.])
Point(1.0, 2.0)

julia> Point(@SVector [1, 2])
Point(1, 2)

julia> Point(@SVector [1., 2.])
Point(1.0, 2.0)

julia> Point(@SVector [1f0, 2f0])
Point(1.0f0, 2.0f0)
```

> What we really want is a more specific method that dispatches on Integer and converts to Float64 

May be want you is something like. 
```suggestion
Point(coords::Integer...) = Point(float(SVector(coords...)))
```

Reply:
Consider now that someone types `Point(1°, 2°)` or any instance of a more complicated type that is not `Number`, we don't want to convert that to float. Makes sense? That is why we need to leave the existing constructors untouched and make sure that we have an additional method to catch the Integer literal case. Right now you converting anything that is passed in to float.